src/components/editMeasure/populationCriteria/MultipleSelectDropDown.tsx

```diff
@@ -56,6 +56,16 @@ const MultipleSelectDropDown = ({
   onClose,
   ...rest
 }) => {
+  const requiredLabelReadable = (
+    <span style={{ color: "white", fontSize: "1px" }}>required</span>
+  );
```

Comment:
can we use this instead` const requiredLabelReadable = <span className="sr-only">required</span>;`

Reply:
Are we able to use `className` prop in our app ? I need to revisit tailwind and twin.macro setup once again, I forgot. but anyways good to know that Tailwind has this class.

Reply:
I think we have all sorts of CSS styling approaches in our app not just tailwind. I don't think `sr-only` is from tailwind. It is from design systems

Reply:
ooh There is sr-only in Tailwind as well, but yeah I see your point then, Even className should work fine.

Reply:
I didn't know about this!